### ISSUE TITLE
Error about duplicate properties in manifest.json

### ISSUE THREAD

**Issue by kumar303** [mozilla/addons-linter#803]
The linter should show a warning (or error?) if a manifest has duplicate properties in it. This will lead to confusing results when the JSON is parsed by Firefox (I think the last one wins). Example:
````json
{
  "name": "My Extension",
  "name": "My Extension 2",
  "version": "1.0"
}
````

**Comment by tofumatt** [comment]
Oh, good call. I think a straight up error is appropriate here.

- tofumatt

**Comment by tofumatt** [comment]
Doing some research it seems this is totally _allowed_ by the JSON spec (though not recommended, it is technically allowed) and most parsers (PHP, node, Python) just replace the value of the key whenever it hits a new one.

That said, we can check for duplicate keys pretty easily with [`JSON.parse`'s reviver parameter](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/JSON/parse#Using_the_reviver_parameter), so I'll get this done today! 😄

**Comment by ValentinaPC** [comment]
I was able to upload WebExtensions with duplicate keys in manifest, without any issues:
- a WebExtension with 2 "name" keys (the name of the WebExtension after submissions was the second)
- a WebExtension with 2 "versions" keys (the version of the WebExtension after submission was the second aka the last one)

Verified using FF47 and FF50 (Win 7).
Screenshots: 
![2016-07-05_1055](https://cloud.githubusercontent.com/assets/15685960/16577917/7e4edb78-42a0-11e6-855a-19a988c7404b.png)
![2016-07-05_1108](https://cloud.githubusercontent.com/assets/15685960/16577992/e05b7b5a-42a0-11e6-906b-7eb752b08d16.png)

**Comment by muffinresearch** [comment]
@ValentinaPC This might not be released yet. When a patch lands in addons-linter it takes a package release and a lib bump in addons-server to be at a point ready to test.

**Comment by ValentinaPC** [comment]
I saw this closed... so... I've tested! :smile:
